### PR TITLE
Betanetstandard

### PR DESCRIPTION
**Pull request by jackfoxy**
Dotnet SDK projects
Drop NUnit and make the test project Expecto tests
Additional console app using Argu lib
All solution files accessible from within solution IDE
Latest FSharp.Formatting prerelease

### CODE REVIEW

**Comment by enovales** [comment]
Also, you'll want to update the `.gitignore` to include the `docs` directory, now that they are being generated from `docsrc`.

File: build.template [hunk 1]
```diff
@@ -47,8 +47,8 @@ let solutionFile  = "##ProjectName##.sln"
 // Default target configuration
 let configuration = "Release"
 
-// Pattern specifying assemblies to be tested using NUnit
-let testAssemblies = "tests/**/bin" </> configuration </> "*Tests*.dll"
+// Pattern specifying assemblies to be tested using Expecto
+let testAssemblies = "tests/**/bin" </> configuration </> "*Tests.exe"
```
[the remark this answers is not in the document]
Comment (jackfoxy) [reply build.template]: thanks, I fixed this

**Comment by jackfoxy** [comment]
@enovales the point of docs being generated into the docs directory is to populate the github.io pages. Adding docs to the .gitignore would prevent update of the pages.

There was an issue thread somewhere of people who objected to this and wanted to restore the gh-pages orphan branch method of github.io maintenance. 

Ideally there would be an init option for either method.

**Comment by enovales** [comment]
Ah, I see -- nevermind, I definitely prefer this to the gh-pages method!

**Comment by enovales** [comment]
One other small issue I ran into -- the `appveyor.yml` needs to specify the VS 2017 image ([example](https://www.appveyor.com/docs/build-configuration/#specializing-matrix-job-configuration)):
```
    image:
      - Visual Studio 2017
```

I just put it at the top of the `appveyor.yml`.

https://github.com/fsprojects/ProjectScaffold/blob/3f2aedd142e766bebb76d67189829b4187b44f2d/appveyor.yml#L1

After making that change, my appveyor build worked.

**Pull request merged**